pantab/_compat.py

```diff
@@ -1,8 +1,8 @@
-from distutils.version import LooseVersion
+from pandas.util.version import parse
```

Comment:
Looks like CI also wants you to put this after the pandas import

Reply:
Refactored to pull from existing `import pandas as pd` import.